include/nccl_ofi_rdma.h

```diff
@@ -834,11 +834,21 @@ typedef struct nccl_net_ofi_rdma_device {
 #endif
 } nccl_net_ofi_rdma_device_t;
 
+
+struct nccl_net_ofi_rdma_plugin_t {
+	nccl_net_ofi_plugin_t base;
+
+	nccl_ofi_topo_t *topo;
+};
+typedef struct nccl_net_ofi_rdma_plugin_t nccl_net_ofi_rdma_plugin_t;
```

Comment:
Is it intentional that the typedef shadows the structure?

Reply:
It was intentional since it's two different namespaces in C.  But that's going to screw up C++, isn't it?

Reply:
no, this is fine, I just hate how it breaks ctags.

Reply:
fair enough; changed.